FESS-GUI/flywheeloperation.cpp

```diff
@@ -177,90 +178,93 @@ void FlywheelOperation::setDefaults()
+                    {
+                        if (udy_buffer.size() > udy_buffer_limit) udy_buffer.pop();
+                        udy_buffer.push(rx_packet.data_value);
+                        break;
+                    }
+
+                    case ROTATIONAL_POSITION_X:
+                    {
+                        if (rpx_buffer.size() > rpx_buffer_limit) rpx_buffer.pop();
+                        rpx_buffer.push(rx_packet.data_value);
+                        break;
+                    }
+
+                    case ROTATIONAL_POSITION_Y:
+                    {
+                        if (rpy_buffer.size() > rpy_buffer_limit) rpy_buffer.pop();
+                        rpy_buffer.push(rx_packet.data_value);
+                        break;
+                    }
+
+                    default: break;
                 }
-
-                case JERK:
-                {
-                    if (jer_buffer.size() > jer_buffer_limit) jer_buffer.pop();
-                    jer_buffer.push(rx_packet.data_value);
-                    break;
-                }
-
-                case LOWER_DISPLACMENT_X:
-                {
-                    if (ldx_buffer.size() > ldx_buffer_limit) ldx_buffer.pop();
-                    ldx_buffer.push(rx_packet.data_value);
-                    break;
-                }
-
-                case LOWER_DISPLACMENT_Y:
-                {
-                    if (ldy_buffer.size() > ldy_buffer_limit) ldy_buffer.pop();
-                    ldy_buffer.push(rx_packet.data_value);
-                    break;
-                }
-
-                case UPPER_DISPLACMENT_X:
-                {
-                    if (udx_buffer.size() > udx_buffer_limit) udx_buffer.pop();
-                    udx_buffer.push(rx_packet.data_value);
-                    break;
-                }
-
-                case UPPER_DISPLACMENT_Y:
-                {
-                    if (udy_buffer.size() > udy_buffer_limit) udy_buffer.pop();
-                    udy_buffer.push(rx_packet.data_value);
-                    break;
-                }
-
-                case ROTATIONAL_POSITION_X:
-                {
-                    if (rpx_buffer.size() > rpx_buffer_limit) rpx_buffer.pop();
-                    rpx_buffer.push(rx_packet.data_value);
-                    break;
-                }
-
-                case ROTATIONAL_POSITION_Y:
-                {
-                    if (rpy_buffer.size() > rpy_buffer_limit) rpy_buffer.pop();
-                    rpy_buffer.push(rx_packet.data_value);
-                    break;
-                }
-
-                default: break;
             }
-        }
 
-       // if (( emergency_retries > 0 )&&( emergency_acknowlegded == false ))
-       // {
-       //     communication_device->pushCommand(COMMAND_SET_EMER_STOP);
-       //     communication_device->sync();
-       //     emergency_retries--;
-       // }
+           // if (( emergency_retries > 0 )&&( emergency_acknowlegded == false ))
```

Comment:
Is there a todo here? It's generally not a good idea to merge commented code to master